src/components/Jokers/FiftyFifty/FiftyFiftyJoker.js

```diff
@@ -0,0 +1,50 @@
+import { useDispatch, useSelector } from "react-redux";
+
+import {
+  updateIncorrectAnswers,
+  applyFiftyFifty,
+} from "store/slices/gamePlaySlice";
+
+import styles from "./FiftyFiftyJoker.module.css";
+
+const FiftyFiftyJoker = () => {
+  const dispatch = useDispatch();
+
+  const {
+    correctAnswer,
+    incorrectAnswers,
+    fiftyFiftyJoker,
+    answeredQuestionsCount,
+  } = useSelector((state) => state.gamePlay);
+
+  const allAnswers = [...incorrectAnswers, correctAnswer];
+
+  const handleFiftyFifty = () => {
+    if (incorrectAnswers.length < allAnswers.length / 2) {
+      return;
+    }
+
+    let reducedIncorrectAnswers = [...incorrectAnswers];
```

Comment:
Again unnecessary reassignment

Reply:
Without this assignment, if we replace the occurrence of reducedIncorrectAnswers with just incorrectAnswers, an error occurs in the for-loop:
Cannot assign to read only property '1' of object '[object Array]'